images/recycler/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# This is the default deployment strategy image for OpenShift Origin. It expects a set of
+# environment variables to parameterize the deploy:
+#
+#   KUBERNETES_MASTER - the address of the OpenShift master
+#   KUBERNETES_DEPLOYMENT_ID - the deployment identifier that is running this build
+#
+# The standard name for this image is openshift/origin-deployer
+#
+FROM openshift/origin
```

Comment:
Does this need to be derived from the origin image?  Seems like you basically need the following packages:

1.  Bash
2.  Sudo
3.  Sed
4.  Whatever provides xargs

Do you need coreutils?

My concern is that this image is going to be really big; I would prefer to keep infra images as small as possible.